cpg_infra/billing_aggregator/aggregate/utils.py

```diff
@@ -513,24 +513,46 @@ async def get_jobs_for_batch(
 async def get_batch_by_id(
     batch_id: str,
     token: str,
+    attempts: int = 5,
 ) -> BatchType:
     """
     Get batch details by batch_id
     """
     try:
         url = HAIL_BATCH_API.replace('{batch_id}', batch_id)
-        logger.info(f'Getting batch: {url}')
+        # logger.info(f'Getting batch: {url}')
```

Comment:
You don't like this logging?

Reply:
Was printing out missing batches, so disabled it for my testing. I will put back in.